pkg/services/app/add.go

```diff
@@ -444,6 +527,8 @@ apiVersion: wego.weave.works/v1alpha1
 kind: Application
 metadata:
   name: {{ .AppName }}
+  labels:
+    name: {{ .AppHash }}
```

Comment:
We shouldn't use `name` as the label name. It should be something qualified like:
`weave-gitops.weave.works/app-identifier`